packages/dapp/src/components/Navbar/Menu/ChainsMenu/ChainsMenu.tsx

```diff
@@ -16,25 +15,19 @@ interface NavbarMenuProps {
 }
 
 export const ChainsMenu = ({ handleClose, open }: NavbarMenuProps) => {
-  const i18Path = 'navbar.walletMenu.';
+  const I18_PATH = 'navbar.walletMenu.';
   const { t: translate } = useTranslation();
   const chains = useChainsContent();
   const theme = useTheme();
   const { account } = useWallet();
-  const [openNavbarChainsMenu, onOpenNavbarChainsMenu, onOpenNavbarSubMenu] =
-    useMenuStore(
-      (state) => [
-        state.openNavbarChainsMenu,
-        state.onOpenNavbarChainsMenu,
-        state.onOpenNavbarSubMenu,
-      ],
-      shallow,
-    );
+  const [openNavbarChainsMenu, onOpenNavbarChainsMenu] = useMenuStore(
+    (state) => [state.openNavbarChainsMenu, state.onOpenNavbarChainsMenu],
+  );
```

Comment:
Why did we remove `shallow` from here?

Reply:
Because I added the "shallow" within the createWithEquality function which replaces it right here? Correct me, if I am wrong - we need to talk about all that "shallow"s anyways